cyflash/bootload.py

```diff
@@ -368,6 +400,44 @@ def progress(self, message=None, current=None, total=None):
             self.out.write("\r%s (%d/%d)" % (message, current, total))
         self.out.flush()
 
+def stopbits_convert(value):
+    import serial
+    if value=="1":
+        stopbits = serial.STOPBITS_ONE
+    elif value=="1.5":
+        stopbits = serial.STOPBITS_ONE_POINT_FIVE
+    elif value=="2":
+        stopbits = serial.STOPBITS_TWO
+    else:
+        stopbits = serial.STOPBITS_ONE
```

Comment:
this could be simplified as:
```
mapping = {"1": serial.STOPBITS_ONE,
           "1.5": serial.STOPBITS_ONE_POINT_FIVE,
           ...
           }
if not value in stopbits:
    print '\nillegal arugment', value, 'for stopbit using', stopbits, 'instead\n'
stopbits = mapping.get(value, serial.STOPBITS_ONE)
```